tools/pubchem/pubchem_fetch.xml

```diff
@@ -0,0 +1,62 @@
+<tool id="pubchem_fetch" name="Pubchem Fetch" version="0.1">
+    <requirements>
+        <requirement type="package">tar</requirement>
+        <requirement type="package">wget</requirement>
+    </requirements>
+
+    <command detect_errors="exit_code"><![CDATA[
+        sh $fetch
```

Comment:
This can be any shell, this can be dash, zsh, ksh or whatever implementation the execution node is using. It is better to be explicit.